### PR TITLE
UI:  update Ember to 3.28.6

### PR DESCRIPTION
**Pull request by valeriia-ruban**
### Description

Update Ember version from 3.27 to 3.28. 

### Testing & Reproduction steps

- run make ui-docker to build the ui image
- run make dev-docker
- clone https://github.com/WenInCode/consul-setup
- in the consul-setup repo go to the setups/peering directory
- run docker-compose up
- run yarn setup:peerings

### Links
https://hashicorp.atlassian.net/browse/CC-4232

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] not a security concern

### CODE REVIEW

File: ui/GNUmakefile [hunk 1]
```diff
@@ -25,6 +25,8 @@ deps: clean
 	cd packages/consul-ui && \
 		$(MAKE) deps
 
+display-node-version:
+	node --version
```
Comment (consul-ui-services) [inline ui/GNUmakefile:29]: @valeriia-ruban can we remove this  and the call to it on line 40. This was left in from when I was testing what was going on with Vercel builds failing.

**Comment by WenInCode** [comment]
The peering create generate token tab's copy token button doesn't work as expected in this update. I compared it on master on both real build and just running the local ember server afterwards. Doesn't work as expected with new ember, but it does on main. 

## Steps to Reproduce

1. run the local server and enable peering with `CONSUL_PEERINGS_ENABLE` 
2. generate a token on the peerings page
3. Click either of the copy buttons 
4. Paste in the console, or wherever you can paste text.

### Expected
The code that was supposed to be copied is pasted 

### Actual
It doesn't paste the code you intended on copying

Screenshot 2023-03-20 at 12 06 48 PM

I will also try to figure out why this is happening

**Comment by WenInCode** [comment]
I'm wondering if it has something to do with it being inside a modal. The other copy buttons outside of modals seem to work fine.

**Review by WenInCode — APPROVED** [review]
Tried it again on a build. LGTM!

**Pull request merged**